github3/pulls.py

```diff
@@ -254,6 +256,9 @@ def is_merged(self):
 
         :returns: bool
         """
+        if self.merged:
```

Comment:
So the name is confusing if this is the first part of github3.py with which you interact with. Everything else does not retrieve a specific attribute and the `merged` attribute is a new addition to what the API returns. (Hence why it was missing and `is_merged` was not a bad name choice at the time.)

The pattern with the library is that attributes do not retrieve data for you or make API calls, but methods do. It's a logical distinction.

I would say someone can rely on `pr.merged` existing but they need to check that it's actually a boolean instead of just using `is pr.merged`. That said, I'm still conflicted about short-circuiting the method. I can't think of a place we do that elsewhere but I'm also in favor of saving people a request against their ratelimit.